README.md

```diff
@@ -81,6 +81,36 @@ Entries that are nested within another Entry. Only Categories or Sub-Categories
 }
 ```
 
+### Deprecated node mapping
+When breaking changes such as deletion/collapsing of IDs or moving to a different category occur, the `deprecated_node_mapping` will serve as a reference to find the latest ids for deprecated nodes.
+
+**Any time such a breaking change occurs, it requires a new entry to be added to the deprecated_node_mapping**
+
+#### Format
+```json
+{
+  "deprecated_id": {
+    "1.3": "new_id",
+    "2.0": "newest_id"
+  }
+}
+```
+
+#### Example
+```json
+{
+  "unvalidated_redirects_and_forwards.open_redirect.get_based_all_users": {
+    "2.1": "unvalidated_redirects_and_forwards.open_redirect.get_based"
+  },
+  "unvalidated_redirects_and_forwards.open_redirect.get_based_authenticated": {
+    "2.1": "unvalidated_redirects_and_forwards.open_redirect.get_based"
+  },
+  "unvalidated_redirects_and_forwards.open_redirect.get_based_unauthenticated": {
+    "2.1": "unvalidated_redirects_and_forwards.open_redirect.get_based"
```

Comment:
up to you, but I thought the full example helps describe how multiple nodes are being collapsed into one.